snmp/Dockerfile

```diff
@@ -10,5 +13,6 @@ RUN pip install snmpsim
 RUN groupadd snmpsim && useradd snmpsim -g snmpsim
 
 VOLUME /usr/snmpsim/data
+USER snmpsim
```

Comment:
I was running into permission issues rebuilding the image maybe something has changed in the `snmpsim` library since the time this image was first build.